core/src/test/java/org/jdbi/v3/core/statement/TestPreparedBatchGenerateKeysPostgres.java

```diff
@@ -85,29 +88,34 @@ public void testBatchResultBearing() {
             List<IdCreateTime> ids = batch1.executePreparedBatch("id", "create_time")
                 .map((r, ctx) -> new IdCreateTime(
                     r.getInt("id"),
-                    r.getTimestamp("create_time")))
+                    r.getObject("create_time", OffsetDateTime.class)))
                 .list();
 
-            PreparedBatch batch2 = h.prepareBatch("update something set create_time = now() where name like :name returning id, name, create_time");
+            var now = OffsetDateTime.ofInstant(Instant.ofEpochSecond(1234567), ZoneOffset.UTC);
```

Comment:
consider using `OffsetDateTime.now(clock)` with a clock that has a constant time.